packages/cli/src/cmds/add.ts

```diff
@@ -224,25 +224,21 @@ export default class Add extends YargsCommand {
   public readonly description = "Adds features to your Teams application.";
 
   public readonly subCommands: YargsCommand[] = [
-    ...(isPreviewFeaturesEnabled()
-      ? [
-          // Category 1: Add Teams Capability
-          ...(isBotNotificationEnabled()
-            ? [new CapabilityAddNotification(), new CapabilityAddCommandAndResponse()]
-            : []),
-          ...(isWorkflowBotEnabled() ? [new CapabilityAddWorkflow()] : []),
-          new CapabilityAddSSOTab(),
-          new CapabilityAddTab(),
-          new CapabilityAddBot(),
-          new CapabilityAddMessageExtension(),
+    // Category 1: Add Teams Capability
+    new CapabilityAddNotification(),
+    new CapabilityAddCommandAndResponse(),
+    ...(isWorkflowBotEnabled() ? [new CapabilityAddWorkflow()] : []),
+    new CapabilityAddSSOTab(),
+    new CapabilityAddTab(),
+    ...(isSPFxMultiTabEnabled() ? [new CapabilityAddSPFxTab()] : []),
```

Comment:
What will happen if user execute add spfx under azure project?

Reply:
Open a bug because the behaviors to add features are also abnormal. https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/15554342